<launch>
    <rosparam file="$(find leap_motion)/config/listener_params.yaml" command="load"/>
    <node pkg="leap_motion" type="leap_motion_driver_node" name="leap_driver" output="screen" launch-prefix="valgrind"/>
    <rosparam file="$(find leap_motion)/config/filter_params.yaml" command="load"/>
    <!--node pkg="leap_motion" type="leap_motion_filter_node" name="leap_filter" output="screen"/-->
</launch>
